app/controllers/api/users_controller.rb

```diff
@@ -2,98 +2,64 @@ class Api::UsersController < ApplicationController
   skip_before_action :authenticate_request, only: %i[login register]
 
   def login
-    # TODO: Don't use user params directly, always sanitize user input!
-    # Here: use user_params instead of params
-    authenticate params[:name], params[:password]
+    authenticate(user_params[:name], user_params[:password])
   end
 
-	def index
-    # TODO: I guess this is a placeholder for now, right?
-		i = {"user" => "Hello world!"}
-		render json: i
-	end
-
   def register
     @user = User.create(user_params)
     if @user.save
-      response = { message: 'User created successfully' }
-      render json: {
-        message: response,
-        # TODO: What happens, if the AuthenticateUser call fails?
-        access_token: AuthenticateUser.new(user_params[:name], user_params[:password]).call
-      },
-      status: :created
+      authenticate(user_params[:name], user_params[:password], 'User created successfully')
     else
       render json: @user.errors, status: :bad
     end
   end
 
-  # TODO: This method should be private
-  def user_params
-    params.permit(
-      :name,
-      :password
-    )
+  def chatroom
+    if @current_user.active == 2
+      @chatroom = @current_user.chatrooms.order("created_at desc").first
+      @user = @chatroom.users.where.not(id: @current_user.id)[0]
```

Comment:
Better: call the other user @other, also use .first to access the user (instead of [0])